pkg/member/member.go

```diff
@@ -369,12 +428,12 @@ func (m *Member) GetMemberLeaderPriority(id uint64) (int, error) {
 	return int(priority), nil
 }
 
-func (m *Member) getMemberBinaryDeployPath(id uint64) string {
+func (m *EmbeddedEtcdMember) getMemberBinaryDeployPath(id uint64) string {
 	return path.Join(m.rootPath, fmt.Sprintf("member/%d/deploy_path", id))
 }
 
 // GetMemberDeployPath loads a member's binary deploy path.
-func (m *Member) GetMemberDeployPath(id uint64) (string, error) {
+func (m *EmbeddedEtcdMember) GetMemberDeployPath(id uint64) (string, error) {
```

Comment:
I'm not sure why set so many functions to return error but not handle error.

Reply:
no regression caused by this pr. we can separately address this concern generally in the entire code base later.